### ISSUE TITLE
Feature request - add comment style to eslint.codeAction.disableRuleComment

### ISSUE THREAD

**Issue by OneHoopyFrood** [microsoft/vscode-eslint#1490]
## TLDR:
I'd like a setting added to eslint.codeAction.disableRuleComment to have it inject block-style comments (`/* */`) rather than line style (`//`)

## Context: 
I use [eslint-require-description](https://mysticatea.github.io/eslint-plugin-eslint-comments/rules/require-description.html) to enforce the use of comments for eslint-ignore comments. (See https://github.com/eslint/rfcs/pull/33).

This helps us reduce abuse of ignore comments and to aid in code review. We also have a formatting rule that limits comment line length and wraps them.

The automatic injection of ignore rules is very useful, but when adding descriptions to said ignore comments I nearly _always_ overrun our line length and have to convert the comment to a block style. This does look nicer and I don't want to disable this feature. 
It would be very nice to be able to have a setting to have the ignore rule use block-style comments. The most logical place I see to attach such a setting is to eslint.codeAction.disableRuleComment.

**Comment by MariaSolOs** [comment]
I think we can close this now @dbaeumer :)
@OneHoopyFrood see #1499 for info on using the new setting!

**Comment by dbaeumer** [comment]
@MariaSolOs thanks for pointing this out.

Closing the issue.
